yt/chyt/controller/internal/api/api.go

```diff
@@ -840,3 +840,23 @@ func (a *API) SetSecrets(ctx context.Context, alias string, secrets map[string]a
 	)
 	return err
 }
+
+func (a *API) Resume(ctx context.Context, alias string) error {
+	if err := a.CheckExistence(ctx, alias, true /*shouldExist*/); err != nil {
+		return err
+	}
+	if err := a.CheckPermissionToOp(ctx, alias, yt.PermissionManage); err != nil {
+		return err
+	} // TODO: maybe yt.PermissionUse?
+
+	oplet, err := a.getOpletFromCypress(ctx, alias, nil, a.cfg.AgentInfo)
+	if err != nil {
+		return err
+	}
+
+	if err = oplet.ResumeOp(ctx, "api call"); err != nil {
+		return err
+	}
+	err = oplet.EnsurePersistentStateFlushed(ctx)
+	return err
```

Comment:
I thought one more time about it, and now I think it's not a good idea since it can start or abort an operation straight from API - something that never happened before, AFAIK